tests/server/testServerWithStorage.js

```diff
@@ -338,9 +304,54 @@ sjrk.storyTelling.server.testServerWithStorageDefs = [{
                     files: {
                         "file": "{testCaseHolder}.options.testUploadOptions.testPNGFile"
                     }
-                },
+                }
+            }
+        }
+    },
+    dynamicComponents: {
+        replaceFile: {
```

Comment:
One issue I think should have been addressed in @BlueSlug's SJRK-289 pull request, which has already been merged, is when re-uploading a media file, the previous media file should be removed from the the /uploads directory, so the server doesn't end with many stray images. 

However, this issue still exists here. Is there a changed plan to address it?

Reply:
We needed to keep the original files because when editing a published story you could delete a published file. If the story being edited isn't yet published, the currently published story would look broken because the file is missing. 

I've filed a new issue, [SRJK-431](https://issues.fluidproject.org/browse/SJRK-431), for automatically cleanup orphaned files.